src/extension.ts

```diff
@@ -318,6 +332,20 @@ export function activate(context: vscode.ExtensionContext): void {
     registerCommand('appService.AddCosmosDBConnection', addCosmosDBConnection);
     registerCommand('appService.RemoveCosmosDBConnection', removeCosmosDBConnection);
     registerCommand('appService.RevealConnection', async (node: CosmosDBConnection) => ext.cosmosAPI.revealTreeItem(node.cosmosDBDatabase.treeItemId));
+    registerCommand('appService.ShowDeploymentLogs', async (node: DeploymentTreeItem) => {
+        const parsedLogs: string = await node.getDeploymentLogs();
+        const document: vscode.TextDocument = await vscode.workspace.openTextDocument({
+            content: parsedLogs
+        });
+        await vscode.window.showTextDocument(document);
+    });
+    registerCommand('appService.Redeploy', async (node: DeploymentTreeItem) => await node.redeployDeployment());
+    registerCommand('appService.DisconnectRepo', async (node: DeploymentsTreeItem) => await node.disconnectRepo());
+    registerCommand('appService.ConnectToGitHub', async (node: ConnectToGitHubTreeItem) => {
+        const siteTreeItem: SiteTreeItem = node.parent.parent;
```

Comment:
Again - you don't have to do all this parent stuff to get `siteTreeItem`. Just use the root directly